From e23a80ebbf81be5d993dce4ef90b9463e28602e9 Mon Sep 17 00:00:00 2001
From: Huayi Zhang <irachex@gmail.com>
Date: Sun, 23 Dec 2018 05:29:45 +0800
Subject: [PATCH] Add missing `\` in gpu Makefile

---
 gpu/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gpu/Makefile b/gpu/Makefile
index 47eb3397..bdc254a1 100644
--- a/gpu/Makefile
+++ b/gpu/Makefile
@@ -45,7 +45,7 @@ CUOBJ =     impl/BinaryDistance.o \
             impl/VectorResidual.o \
             GpuDistance.o \
             GpuIndex.o \
-            GpuIndexBinaryFlat.o
+            GpuIndexBinaryFlat.o \
             GpuIndexFlat.o \
             GpuIndexIVF.o \
             GpuIndexIVFFlat.o \
